src/utilities/example-group/example-group.factory.ts

```diff
@@ -8,25 +13,8 @@ class ExampleGroupFactory {
     const parameters = operation.parameters;
     const groupNames = this.getGroupNames(parameters);
 
-    const requiredExamples = parameters
-      .filter((parameter) => parameter.required)
-      .reduce((examples, parameter) => {
-        return {
-          ...examples,
-          [parameter.name]:
-            parameter.example ??
-            parameter.examples?.[DEFAULT_PARAMETER_GROUP]?.value,
-        };
-      }, {});
-
-    const nonRequiredExamplesWithExampleField = parameters
-      .filter((parameter) => !parameter.required && parameter.example)
-      .reduce((examples, parameter) => {
-        return {
-          ...examples,
-          [parameter.name]: parameter.example,
-        };
-      }, {});
+    const requiredExamples = this.getRequiredExamples(parameters);
+    const nonRequiredExamples = this.getNonRequiredExamples(parameters);
```

Comment:
this naming seems confusing. Is there a way to call out that these are just values from the `example` field?

Reply:
From what I can tell, this is just a refactoring to put the logic into a function. It still pulls from both examples and example.

Reply:
nvmd